src/components/LearnerActivityTable/LearnerActivityTable.test.jsx

```diff
@@ -139,6 +139,28 @@ describe('LearnerActivityTable', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('renders empty state correctly if groups have no loaded', () => {
```

Comment:
Do you think it would be an improvement of this test if it stated what the expected behavior is in its name, and then also tested for that? For example testing for `We are currently processing the latest updates` to be in the document?